CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Describes notable changes.
 
+#### 1.21.1 - 2020/01/03
+- Task data is still saved into old data field, to allow seamless upgrade process.
+When upgrade from versions before 1.21.1 is finished, i.e. all cluster nodes have the new version;
+  `TasksProperties.copyDataToTwTaskField` can be set to `false`, stopping writing into the old data field.
+- `TasksProperties.Environment.previousVersion` is mandatory.
```

Comment:
Does this mean `previousVersion` would be required irrespective of whether we want to stop writing to old data or not?
It would be ok to have values beyond 1.20.0 as well for the previousVersion right, like `1.21.0`?

Reply:
Yep, the idea is to specify a version you have in production.

If you forget to keep changing it, it is fine.

Lets say you have 1.25.0 in prod and you are upgrading to 1.26.0 and 1.26.0 can be only upgraded from 1.25.0.

If you forgot 1.20.0 in, you will still get an error, that your version is too old. Then you will change the property and it would work.

Reply:
Sounds good... Thank you! 👍

Reply:
Adding a validity check, that previous version can not be > current version in code may be a good idea. Have to digest it.

Reply:
Yeah... I saw the comment about future checks below ` // Future checks. This version allows to upgrade from any earlier versions.`, so thought you might already have that in mind 😄